### ISSUE TITLE
eslint plugin lit-a11y fails on rule "role-has-required-aria-attrs" with role=option

### ISSUE THREAD

**Issue by atomiro** [open-wc/open-wc#2231]
## Expected behavior
Element with role="option" and aria-selected="${etc}" passes rule "role-has-required-aria-attrs"

## Actual Behavior
lit-a11y throws an error even though both role and aria-selected are set on the element. 

## Context
This is the template it errors on. I've triggered other variants of this rule (such as with role="combobox") and gotten those to pass, but this one still recurs. 
```
html` <li
        @click=${this.selectOption}
        class="px-3 py-1 ${classMap(this.itemHighlighted(index))}"
        role="option"
        aria-selected="${this.itemSelected(option) ? 'true' : 'false'}"
      >
        ${this.itemDisplay(option)}
      </li>`
```

Which packages and versions are you using.
v1.0.1

**Comment by thepassle** [comment]
I was unable to reproduce this locally: 

![image](https://user-images.githubusercontent.com/17054057/122355610-55484680-cf52-11eb-9cfc-2094d9dbf24e.png)

Could you provide some more information, or potentially a minimal reproduction repo?
